blueprints/sitemap.yaml

```diff
@@ -25,6 +25,12 @@ form:
                     type: bool
                     required: false
 
+                header.sitemap.lastmod:
+                  type: datetime
+                  toggleable: true
```

Comment:
Do we really need it to be `toggleable`? Other sitemap options aren't (except for the `ignore` itself) 🤔

Reply:
I thought that since it's an optional field for users which require a manually set date, it makes sense to make it toggleable. So users which use the file modification date don't need this option configured.

Reply:
So if you want to keep `toggleable` here, then you should add another check for it in the _sitemap.php:313_, because even if it's turned off, it still may contain the filled date in the input.

IMO it would be enough to remove the toggle here and probably much better to add a description instead, that "If the field is empty, it will use the default modified date."